ocs_ci/templates/workloads/fio/fill_pool_pod.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fill-pool-pod
+spec:
+  containers:
```

Comment:
instead of pod can we use kind: deployment as it's more reliable

Reply:
Ack

Reply:
I have replaced the Pod Workpload with a Kubernetes Job. This provides better lifecycle management and 
consistency with OpenShift workload standards